bot/cogs/geo_cog/streetviewrandomizer/street_view_random.py

```diff
@@ -0,0 +1,249 @@
+                + f'size={size}&'
+                + f'location={coord.lat},{coord.lon}&'
+                + f"heading={args['headings']}&"
+                + f"pitch={args['pitches']}&"
+                + f"fov={args['fovs']}&"
+                + f"key={bot_secrets.secrets.geocode_key}"),
+                coord.lat,
+                coord.lon,
+                country_iso3,
+                new_country_selections_if_error)
+
+    @staticmethod
+    def compute_area(gdf: gpd.GeoDataFrame) -> gpd.GeoDataFrame:
+        gdf = gdf.eval("AREA = geometry.to_crs('esri:54009').area")
+        gdf = gdf.sort_values(by="AREA", ascending=False)
+
+        gdf = gdf.eval("AREA_PERCENTAGE = AREA / AREA.sum()")
+        gdf = gdf.sort_values(by="AREA_PERCENTAGE", ascending=False)
+        gdf = gdf.reset_index(drop=True)
+
+        return gdf
+
+    # Find a random point in the specified country up to 50 times or until street view is found.
+    async def find_available_image(self, gdf: gpd.GeoDataFrame, radius_m: int) -> ImageData:
+        """
+        Get an image from Google Street View Static API.
+        :param gdf: Geo-dataframe for a single target country.
+        :param radius_m: Grab the nearest streetview within a radius.
+        :return: ImageData dataclass.
+        """
+        coord: Coordinate
+        country_df = None
+        attempts = 0
+        total_elapsed_time_ms = 0
+        image_found = False
+
+        logging.log(1, str(country_df) + "looking in")
+        while not image_found:
+            start = timer()
+            attempts += 1
+            country_df = self.get_random_country(gdf)
+            min_lon, min_lat, max_lon, max_lat = country_df.total_bounds
+            random_lat = random.uniform(min_lat, max_lat)
+            random_lon = random.uniform(min_lon, max_lon)
+
+            random_lat = 27.98812003113945 if isclose(27.98812003113945, random_lat, abs_tol=10**-4) else random_lat
+            random_lon = 86.92497299755392 if isclose(86.92497299755392, random_lon, abs_tol=10**-4) else random_lon
+            radius_m = 1000 if isclose(86.92497299755392, random_lon, abs_tol=10**-4) else radius_m
+
+            logging.log(1, f"\n{attempts}")
+            coord = Coordinate(random_lat, random_lon)
+            status: str = ""
+
+            if attempts >= MAX_ATTEMPTS:
+                logging.warning("\nMaximum safe attempts (50) exceeded. Choosing a different country.")
+                return ImageData(None, None, None, None, ERROR)
+
+            if coord.within(country_df.geometry.values[0]):
+                coord, status = await API.has_image(coord, radius_m)
+
+            end = timer()
+            elapsed_ms = (end - start) * 1000
+            total_elapsed_time_ms += elapsed_ms
+
+            if status == "OK":
+                break
+
+        returnValue: ImageData = ImageData(coord, country_df, attempts, total_elapsed_time_ms, 0)
+        return returnValue
```

Comment:
No need to declare a var and then immediately return it
```suggestion
        return ImageData(coord, country_df, attempts, total_elapsed_time_ms, 0)
```